types/bunyan-slack/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for bunyan-slack 0.0
+// Project: https://github.com/qualitybath/bunyan-slack#readme
+// Definitions by: Eric Chen <https://github.com/fshiori>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare namespace BunyanSlack {
+    interface Options {
+        webhook_url: string;
+        webhookUrl?: string;
+        customFormatter?: any;
+        icon_url?: string;
+        iconUrl?: string;
+        icon_emoji?: string;
+        iconEmoji?: string;
+        channel?: string;
+        username?: string;
+    }
+
+    interface BunyanSlackConstructor extends NodeJS.WritableStream {
```

Comment:
This is likely incorrect,per https://github.com/qualitybath/bunyan-slack/blob/master/lib/bunyan-slack.js. The constructor itself is not likely a writable stream, and the result of the constructor does not seem to be one either.

You may want something like this instead?
```ts
interface BunyanSlack {
  write(record: unknown): void;
}
interface BunyanSlackConstructor {
  new (options: Options, error?: any): BunyanSlack;
}
```

Alternatively, you might want to define it as a class that merges with a namespace per my other comment.